### PR TITLE
Prep for 2.1.1

### PR DESCRIPTION
**Pull request by j2sol**

### CODE REVIEW

File: envs/example/defaults-2.0.yml [hunk 1]
```diff
@@ -565,8 +565,8 @@ common:
     mcelog: False
   ursula_monitoring:
     method: tar # git|tar
-    tar_url: https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/2.1.0.tar.gz
-    tar_version: 2.1.0
+    tar_url: https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/2.1.1.tar.gz
```
Comment (zfeldstein) [inline envs/example/defaults-2.0.yml]: This tar doesn't exist currently otherwise looks good.

Comment (j2sol) [inline envs/example/defaults-2.0.yml]: Yup, I'm waiting for site controller to merge and deploy.

**Comment by zfeldstein** [comment]
👍

**Comment by j2sol** [comment]
retest

File: envs/example/defaults-2.0.yml (hunk 1 above)
Comment (j2sol) [inline envs/example/defaults-2.0.yml]: File is there now.

**Pull request merged**